back/app/models/custom_field.rb

```diff
@@ -231,6 +232,16 @@ def other_option_text_field
     )
   end
 
+  def ordered_options
```

Comment:
I wonder if it could make sense to implement this as a `default_scope`. On the other hand, we might not want to have random ordering when e.g. editing the options.

Reply:
Yes indeed, when editing options we don't ever want the random order